### PR TITLE
updating ONNX batch model generator support scripts for AutoML for Images

### PR DESCRIPTION
**Pull request by vadthyavath**
# PR into Azure/azureml-examples

## Checklist

I have:

- [X] read and followed the contributing guidelines

## Changes

- Updated ONNX batch model generator script for devplatv2
- added onnx yolo utils to the current directory

Yolov5 batch onnx model generator:
[ONNX-batch-model-generation - Microsoft Azure Machine Learning Studio](https://ml.azure.com/runs/silver_coat_mf97rfcn61?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourcegroups/rv-images-rc/workspaces/rv-images-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#outputsAndLogs)
RCNN
[ONNX-batch-model-generation-rcnn - Microsoft Azure Machine Learning Studio](https://ml.azure.com/runs/gray_honey_h7wylklhhq?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourcegroups/rv-images-rc/workspaces/rv-images-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#outputsAndLogs)
Retinanet
[ONNX-batch-model-generation-rcnn - Microsoft Azure Machine Learning Studio](https://ml.azure.com/runs/loyal_feast_ycs4kpc76h?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourcegroups/rv-images-rc/workspaces/rv-images-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#outputsAndLogs)
Segmentation
[ONNX-batch-model-generation-maskrcnn - Microsoft Azure Machine Learning Studio](https://ml.azure.com/runs/green_oxygen_z27jlgg0v3?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourcegroups/rv-images-rc/workspaces/rv-images-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#overview)
fixes #

### CODE REVIEW

File: sdk/jobs/automl-standalone-jobs/ONNX_batch_model_generator_automl_for_images.py [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+        required=False,
+        help="Size of the model from small, medium, large, xlarge",
+    )
+    parser.add_argument(
+        "--box_iou_thresh", type=float, required=False, help="IoU threshold"
+    )
+
+    # parse arguments
+    args = parser.parse_args()
+    experiment = Run.get_context().experiment
```
Comment (PhaniShekhar) [inline sdk/jobs/automl-standalone-jobs/ONNX_batch_model_generator_automl_for_images.py:281]: Why is this changed to using "Run.get_context()" ?
To get the mlflow_tracking_uri, we can create a MLClient using subscription_id, resource_group and workspace.
and get it using

MLFLOW_TRACKING_URI = ml_client.workspaces.get(
    name=ml_client.workspace_name
).mlflow_tracking_uri

Comment (vadthyavath) [reply sdk/jobs/automl-standalone-jobs/ONNX_batch_model_generator_automl_for_images.py:281]: if I don't get it from context, then scriptrunconfig needs to be authenticated manually each time. One customer reported that as well on ONNX. That's why I modified to get_context. I have seen this methodology across lot of scripts in this repo ex: https://github.com/Azure/azureml-examples/blob/a9c5ff547feb2d3c266313da9750b0f0cd0c477c/cli/jobs/pipelines-with-components/rai_pipeline_adult_analyse/components/src_train_logreg/train.py

**Review by PhaniShekhar — APPROVED** [review]

**Review by sharma-riti — APPROVED** [review]
